ga-ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -390,7 +390,16 @@
     "RoleName" : "idam:citizen",
     "Authorisation": "",
     "CaseTypeID": "GENERALAPPLICATION",
-    "AccessProfiles": "citizen-profile",
```

Comment:
can we create a non-prod file for these changes ?

Reply:
Moved to non-prod file